understanding/22/focus-appearance-minimum.html

```diff
@@ -232,7 +232,7 @@ <h4>Unusual shapes and gradients</h4>
                 <p>Some designs have pages with a non-solid background image covering the whole (or part) of the page or make use of parallax scrolling effects which result in a near-infinite number of color combinations if a page is scrolled and/or changes are made to the viewport size.</p>
                 <p>If the contrast of background colors that change are close enough to need to be tested for each combination then they would likely not meet the user need of people with low vision in certain scroll combinations and would likely fail in certain combinations as well. In these cases it would be an easy solution to use a double ring focus indicator or some other mechanism to indicate focus such as a solid box with a border to guarantee there is sufficient contrast across variations of background images or background gradients.</p>
 
-                <p>Some components may be very large, such as editing areas in a Content Management System or code-editor that fill the screen. There maybe some circumstances where a large editing area may not be as useful, however, it may still be beneficial to some. There is no exception for this scenario.</p>
+                <p>Some components may be very large, such as editing areas in a Content Management System or code-editor that fill the screen. There maybe some circumstances where a large focus indicator may not be as useful, however, it may still be beneficial to some. There is no exception for this scenario.</p>
```

Comment:
that `, however,` looks a bit awkward. perhaps `... may not be as useful. However, it may ...` or `... may not be as useful – however, it may ...`